### ISSUE TITLE
Security leak

### ISSUE THREAD

**Issue by asyd** [abriotde/openhems-sample#11]
You should remove this from git (and its history), there are others reference to your configuration

https://github.com/abriotde/openhems-sample/blob/71bdfc3d18c67fff3c95547397dc788ef47b6888/config/openhems_test_emhass.yaml#L4-L5

**Comment by abriotde** [comment]
Yes, I know. I didn't consider it as a real problem as it is only at my home and I can change  it. 
I fix it, on dev, it will be on mail after and I will remove it after.
